exercises/connect/test.ml

```diff
@@ -2,119 +2,64 @@ open OUnit2
 open Connect
 
 let show_player = function
-| Some X -> "X"
-| Some O -> "O"
-| None -> "None"
+  | Some X -> "X"
+  | Some O -> "O"
+  | None -> "None"
 
 let ae exp got = assert_equal ~printer:show_player exp got
 
 let tests = [
   "an empty board has no winner" >::(fun _ctxt ->
-    let board = [
-      ". . . . ."; 
-      " . . . . ."; 
-      "  . . . . ."; 
-      "   . . . . ."; 
-      "    . . . . ."
-      ] 
-    in
-    ae None (connect board)
-  );
+      let board = [". . . . ."; " . . . . ."; "  . . . . ."; "   . . . . ."; "    . . . . ."] 
+      in
+      ae None (connect board)
+    );
   "X can win on a 1x1 board" >::(fun _ctxt ->
-    let board = [
-      "X"
-      ] 
-    in
-    ae (Some X) (connect board)
-  );
+      let board = ["X"] 
+      in
+      ae (Some X) (connect board)
+    );
   "O can win on a 1x1 board" >::(fun _ctxt ->
-    let board = [
-      "O"
-      ] 
-    in
-    ae (Some O) (connect board)
-  );
+      let board = ["O"] 
+      in
+      ae (Some O) (connect board)
+    );
   "only edges does not make a winner" >::(fun _ctxt ->
-    let board = [
-      "O O O X"; 
-      " X . . X"; 
-      "  X . . X"; 
-      "   X O O O"
-      ] 
-    in
-    ae None (connect board)
-  );
+      let board = ["O O O X"; " X . . X"; "  X . . X"; "   X O O O"] 
+      in
+      ae None (connect board)
+    );
   "illegal diagonal does not make a winner" >::(fun _ctxt ->
-    let board = [
-      "X O . ."; 
-      " O X X X"; 
-      "  O X O ."; 
-      "   . O X ."; 
-      "    X X O O"
-      ] 
-    in
-    ae None (connect board)
-  );
+      let board = ["X O . ."; " O X X X"; "  O X O ."; "   . O X ."; "    X X O O"] 
```

Comment:
Is this a forced consequence of `ocp-indent`?

The advantage before was that you could see the drawing of the board.

Reply:
No, this is the default format of lists as emitted by the test-generator. I note we should add a special case for `connect` to inject `\n` after each list item